cmd/flags/admin.go

```diff
@@ -36,8 +36,7 @@ const (
 	adminHTTPPort       = "admin-http-port"
 	adminHTTPHostPort   = "admin.http.host-port"
 
-	healthCheckHTTPPortWarning = "(deprecated, will be removed after 2020-03-15 or in release v1.19.0, whichever is later)"
-	adminHTTPPortWarning       = "(deprecated, will be removed after 2020-06-30 or in release v1.20.0, whichever is later)"
+	adminHTTPPortWarning = "(deprecated, will be removed after 2020-06-30 or in release v1.20.0, whichever is later)"
```

Comment:
please remove both deprecated flags since we're changing this module

Reply:
Removed adminHTTPPortWarning as well. Please review.